html/manifest/file_handlers.json

```
@@ -0,0 +1,44 @@
+{
+  "html": {
+    "manifest": {
+      "file_handlers": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/Manifest/file_handlers",
+          "spec_url": "https://wicg.github.io/manifest-incubations/index.html#file_handlers-member",
+          "support": {
+            "chrome": {
+              "version_added": "102"
+            },
+            "chrome_android": {
+               "version_added": false
```

Sorry, more CI complaints. 

```suggestion
              "version_added": false
```